lib/resources/sample.js

```diff
@@ -0,0 +1,57 @@
+const got = require('got')
+let pptf
+(async function () {
+    try {
+        pptf = (await import('parse-prometheus-text-format')).default
+    } catch (err) {
+        console.log(err)
+    }
+})();
+
+
+let lastCPUTime = 0
+
+async function sampleResources (url, time) {
```

Comment:
`time` appears unused in this function

Reply:
Time is used on line 41, it is the time between samples to determine the average CPU usage